prometheus/README.md

```diff
@@ -32,60 +32,41 @@ please see Install Grafana
+## Install Prometheus and the Prometheus Operator
+
+1. Create the Prometheus objects.
 
-1. Clone the StorageOS deploy repository and move into the
-   `prometheus-operator` directory
-    ```bash
-    git clone https://github.com/coreos/prometheus-operator.git prometheus-operator
-    ```
-1. Deploy the quick start `bundle.yaml`
-    ```bash
-    kubectl create -f prometheus-operator/bundle.yaml
-    ```
-1. Verify that the Prometheus operator is running.
    ```bash
-   kubectl get pods -l apps.kubernetes.io/name=prometheus-operator
+   ./install-prometheus.sh
    ```
 
-## Install Prometheus
+1. Confirm Prometheus is up and running.
 
-Now that the Prometheus Operator is installed, a Prometheus CR can be created
-which the Prometheus operator will act upon to configure a Prometheus StatefulSet
-
-1. Clone the StorageOS deploy repo
-   ```bash
-   git clone https://github.com/storageos/use-cases.git storageos-usecases
-   cd storageos-usecases/prometheus/manifests/prometheus
-   ```
-1. If your cluster uses RBAC then create the necessary Cluster role and service
-   account for Prometheus.
    ```bash
-   kubectl create -f prometheus-rbac.yaml
+   $ kubectl get pods -w -l app=prometheus
+   NAME                                READY   STATUS              RESTARTS   AGE
+   prometheus-prometheus-storageos-0   3/3     READY               0          1m
    ```
-1. Create a Prometheus CR that defines a Prometheus StatefulSet.
-   ```bash
-   kubectl create -f prometheus-cr.yaml
-   ```
-1. Create a ServiceMonitor CR that directs Prometheus to scrape the Endpoints
-   defined in the `storageos` Endpoints resource. Prometheus will scrape the
-   `/metrics` URL of the Endpoints and collect the metrics.
-   ```
-   kubectl create -f storageos-serviceMonitor.yaml
-   ```
-> N.B. An additional serviceMonitor manifest is avaliable that will monitor
-> etcd pods in the etcd namespace. This serviceMonitor can be used with the [etcd running as
-> pods](https://grafana.com/dashboards/10323) Grafana dashboard.
 
-1. In order to view the Prometheus UI in the browser port forward the local
-   port to the Prometheus pod port.
+1. You can see the created PVC using.
+    ```bash
+    $ kubectl get pvc
+    NAME                                     STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS           AGE
+    data-prometheus-prometheus-storageos-0   Bound    pvc-b6c17c0a-e76b-4a0b-8fc6-46c0e1629210   1Gi        RWO            storageos-replicated   65m
+    ```
+
+1. In the Prometheus deployment script, a service monitors is also created. The new Prometheus instance will use the storageos-etcd service monitor to start scraping metrics from the storageos-etcd pods. (Assuming the storageos cluster was setup using ETCD as pods) For more information about service monitors, have a look at the upstream [documentation](https://coreos.com/operators/prometheus/docs/latest/user-guides/getting-started.html)
```

Comment:
Same comment as for docs PR - can we wrap this, and all other lines at 79 chars please. Also, missing opening quote for code section.